ui/components/app/modals/contract-details-modal/contract-details-modal.js

```diff
@@ -124,7 +124,7 @@ export default function ContractDetailsModal({
                   <Text
                     variant={TextVariant.bodySm}
                     as="h6"
-                    display={DISPLAY.FLEX}
+                    display={Display.Flex}
                     color={TextColor.textAlternative}
                     marginTop={0}
```

Comment:
please check this too

Reply:
Nice catch, I don't think `marginTop={0}` is needed here and can be removed

Reply:
I'll remove all such instances in the file included in the PR?

Reply:
I think that makes sense no `Text` component should have a `boxProps` prop and unless there are styles that need to be overridden `margin`, `marginTop`, `marginBottom` etc props with 0 values are redundant because the `Text` component doesn't have any default margins applied to it. It already has 0 margin if that makes sense

Reply:
I have made the changes, could you please review them?